### PR TITLE
Update v1 to 1.0.15

### PR DESCRIPTION
**Pull request by ahmelsayed**
/cc @mhoeger

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -282,6 +282,17 @@
       "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/1.0.3.10182.zip",
       "sha2": "3480E4AC8B8C739AC6BA176468B4CCA4EB2D333D8F1D21CB2A10FEBE2C3C3BF1",
       "FUNCTIONS_EXTENSION_VERSION": "~1"
+    },
+    "1.4.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.15",
```
Comment (soninaren) [inline cli-feed-v3.json]: nit: Latest Net SDK is 1.0.19, may not make a difference since it doesn't have any updates for v1

Comment (ahmelsayed) [reply cli-feed-v3.json]: is it safe to update this to 1.0.19?

Comment (ahmelsayed) [reply cli-feed-v3.json]: nevermind, I realized I uped it to a random version. Updated to 1.0.19. Thanks for catching that.

**Comment by soninaren** [comment]
Forgot to mention, you did need to update the v1 tag, may be update v1-prerelease and we can do an OGF run to make sure it is okay to move to 1.0.19 for v1

**Comment by ahmelsayed** [comment]
🤦‍♂️ good point :) I'll send another PR

**Pull request merged**